EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -204,6 +207,20 @@ class EarthAsDMAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResources>
   std::vector<float> muon_dtSeg_eta_;
   std::vector<float> muon_dtSeg_phi_;
 
+  std::vector<float> rSquaredValues_;
+  std::vector<float> dtGlobalPointYValues;
+  std::vector<float> t0timingValues;
+  std::vector<float> dtGlobalPointYValues_Ztiming;
+  std::vector<float> t0_Z_timingValues;
+
+  std::vector<float> PearsonValues_;
+  std::vector<float> PearsonValues_Ztiming_;
+
+  std::vector<int> stationVector_;
+  std::vector<int> sectorVector_;
+  std::vector<std::pair<int, int>> stationSectorVector_;
+  std::vector<std::pair<int, int>> stationSectorVector_Z_;
```

Comment:
start with a part that determines what level this is on, i.e. `muon_rSquared_`, naming something Values is also not very helpful, everything is a value :D